languages/english.js

```diff
@@ -117,7 +117,7 @@ const ui = {
 
 const warnings = {
   disable_scarf: "This dependency could collect data against your will so think to disable it with the env var: SCARF_ANALYTICS",
-  keylogging: "This dependency can retrieve your keyboard and mouse inputs. It can be used for 'keylogging' attacks/malwares."
+  keylogging: "This dependency can retrieve your keyboard and mouse inputs. It can be used for 'keylogging' attacks/malwares.",
```

Comment:
This change should not be needed, right ?

Reply:
Is done